airbyte-commons-protocol/src/main/java/io/airbyte/commons/protocol/migrations/ConfiguredAirbyteCatalogMigrationV1.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.commons.protocol.migrations;
+
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.commons.version.AirbyteProtocolVersion;
+import io.airbyte.commons.version.Version;
+import io.airbyte.protocol.models.v0.ConfiguredAirbyteCatalog;
+import jakarta.inject.Singleton;
+
+@Singleton
+public class ConfiguredAirbyteCatalogMigrationV1
+    implements ConfiguredAirbyteCatalogMigration<ConfiguredAirbyteCatalog, io.airbyte.protocol.models.ConfiguredAirbyteCatalog> {
+
+  @Override
+  public ConfiguredAirbyteCatalog downgrade(io.airbyte.protocol.models.ConfiguredAirbyteCatalog message) {
+    return Jsons.object(Jsons.jsonNode(message), ConfiguredAirbyteCatalog.class);
+  }
+
+  @Override
+  public io.airbyte.protocol.models.ConfiguredAirbyteCatalog upgrade(ConfiguredAirbyteCatalog message) {
+    return Jsons.object(Jsons.jsonNode(message), io.airbyte.protocol.models.ConfiguredAirbyteCatalog.class);
```

Comment:
trying to understand what's going on here, is `io.airbyte.protocol.models.ConfiguredAirbyteCatalog` identical to the latest `io.airbyte.protocol.models.v1.ConfiguredAirbyteCatalog`? Should anyone be using `io.airbyte.protocol.models.v1.ConfiguredAirbyteCatalog` directly outside of migration or should people always be using `io.airbyte.protocol.models.ConfiguredAirbyteCatalog`?

Reply:
This is currently a transition phase. Eventually, we should be using `io.airbyte.protocol.models.vX` and the objects without an explicit versions will be removed.